src/wp-includes/class-wp-locale.php

```diff
@@ -104,6 +104,16 @@ class WP_Locale {
 	 */
 	public $list_item_separator;
 
+	/**
+	 * The word count type of the locale language.
+	 *
+	 * Default is 'words'.
+	 *
+	 * @since 6.1.0
```

Comment:
Don't think the new issue with added in 6.1 milestone as beta 2 is already released. 🤔

Reply:
Sure, it’s really not intended to be a fixed number, consider it as a placeholder to be updated accordingly